tools/scoring/score/dimensions/dimension.py

```diff
@@ -191,9 +195,10 @@ def incorrect_total(self) -> int:
   def _condense_translations(self, file_type: FileType):
     """Combines translations for all devices within the dictionary."""
     return [
-        matched_translations[file_type]
-        for matched_translations in self.translations.values()
-        if matched_translations[file_type]
+        field
```

Comment:
can you changed to a normal loop please? these are hard to debug and maintaining the code.

Reply:
https://github.com/google/digitalbuildings/pull/527/commits/28747f5ba6a8d44f111a87dc145229ab3dc28e37